web/packages/design/src/theme/theme.ts

```diff
@@ -31,27 +31,21 @@ const space = [0, 4, 8, 16, 24, 32, 40, 48, 56, 64, 72, 80];
 const contrastThreshold = 3;
 
 const colors = {
-  accent: '#651FFF',
-  progressBarColor: '#00BFA5',
-
-  dark: '#000',
+  levels: {
```

Comment:
can you add the same kind of clarifying comment you wrote on your PR description? i thought it was well descripted, and would help if we documented here